mne_features/univariate.py

```diff
@@ -1013,6 +1016,76 @@ def compute_svd_entropy(data, tau=2, emb=10):
     return -np.sum(np.multiply(sv_norm, np.log2(sv_norm)), axis=-1)
 
 
+def compute_powercurve_deviation(sfreq, data, fmin=0.1, fmax=50, 
+                                 with_intercept=True):
+    """ Linear fit to the log-log frequency-curve with fit error (per channel).
+
+    Parameters
+    ----------
+    data : ndarray, shape (n_channels, n_times)
```

Comment:
- The order of the parameters in the docstring should reflect the order in the definition of the function.
- The parameter `with_intercept` is missing from the docstring.

Reply:
Also, I would explain in the docstring what are the MSE (Mean Square Error) and the R2 coefficient (Goodness-of-fit)